### PR TITLE
f-filter-pill@v0.1.0 - Initial component generated

### PR DESCRIPTION
**Pull request by 1itvinka**
### Initial `f-filter-pill` component setup

This is a new atom component used primarily for filter toggles on the search page.

An example of how they currently look:

Screenshot 2021-12-16 at 12 42 57

### CODE REVIEW

File: packages/components/atoms/f-filter-pill/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "test": "vue-cli-service test:unit",
+    "test-component:chrome": "cross-env-shell TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js",
+    "test-a11y:chrome": "cross-env-shell TEST_TYPE=a11y wdio ../../../../wdio-chrome.conf.js",
+    "test:visual": "cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_F_FILTER_PILL} TEST_TYPE=visual percy exec -- wdio ../../../../wdio-chrome.conf.js"
+  },
+  "browserslist": [
+    "extends @justeat/browserslist-config-fozzie"
+  ],
+  "dependencies": {
+    "@justeat/f-services": "1.7.0"
```
Comment (oliversweb) [inline packages/components/atoms/f-filter-pill/package.json]: Will you use this dep?

File: packages/components/atoms/f-filter-pill/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+v0.1.0
+------------------------------
+*December 14, 2021*
+
+### Added
+- Add your change under the following headings: `Added`/`Changed`/`Deprecated`/`Removed`/`Fixed`/`Security`
+!!! Please, add your component to `.circleci/config.yml` under `save_cache_dist_directories` manually after the component is generated.
```
Comment (oliversweb) [inline packages/components/atoms/f-filter-pill/CHANGELOG.md]: You might want to change this.
```suggestion
- Initial scaffolding for new component
```

File: packages/components/atoms/f-filter-pill/package.json (hunk 1 above)
Comment (1itvinka) [reply packages/components/atoms/f-filter-pill/package.json]: not sure yet. I'll remove it for now, and add it later if we need it.

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — APPROVED** [review]

File: packages/components/atoms/f-filter-pill/.eslintignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
Comment (AlixRichards) [inline packages/components/atoms/f-filter-pill/.eslintignore:2]: nit: This file needs a newline at the end, happy for you to add this in your next pr 👍

**Review by AlixRichards — APPROVED** [review]

File: packages/components/atoms/f-filter-pill/.eslintignore (hunk 3 above)
Comment (1itvinka) [reply packages/components/atoms/f-filter-pill/.eslintignore:2]: thanks!

**Pull request merged**